src/main/kotlin/io/snyk/plugin/ui/SnykBalloonNotifications.kt

```diff
@@ -92,6 +101,21 @@ class SnykBalloonNotifications {
             }
         )
 
+        fun showScanningReminder(project: Project) = showInfo(
+            "Scan your project for security vulnerabilities and code issues",
+            project,
+            NotificationAction.createSimpleExpiring("Run scan") {
+                project.service<SnykTaskQueueService>().scan()
+                project.service<SnykAnalyticsService>().logAnalysisIsTriggered(
+                    AnalysisIsTriggered.builder()
+                        .analysisType(getSelectedProducts(getApplicationSettingsStateService()))
```

Comment:
move getApplicationSettingsStateService() invocation inside getSelectedProducts() ?

Reply:
will be done in separate PR